src/test/java/org/primeframework/mvc/BrowserSessionTest.java

```diff
@@ -23,6 +23,24 @@
  * @author Brian Pontarelli
  */
 public class BrowserSessionTest extends PrimeBaseTest {
+  @Test
+  public void not_encrypted_cookie() throws Exception {
```

Comment:
Can we add a "use case" description as the first line inside of this test to better describe the use case you are testing?